content/Software_Projects/Mobile_SDK/App_Registration_Process/index.adoc

```diff
@@ -0,0 +1,78 @@
+== App Registration Process
+All iOS apps must be approved by Yubico and Apple in order to work with the YubiKey 5Ci. Apple requires all iOS apps that communicate with Apple-approved Made for iPhone/iPod/iPad (MFi) devices to be registered with Apple. 
+
+Below you will find the guidelines and process for an Apple application developer to register an iOS app for approval that has completed the Yubico iOS SDK integration.
+
+NOTE: This process is not required for Android applications.
+
+=== Approval Process
+image::iOS_App_Review_Process.png[]
+The <<Actions Required By You,Actions Required By You>> section provides detailed instructions for each step in the approval process. Submitting an app to the App Store without the Yubico 5Ci MFi PPID will result in rejection from Apple.
+
+=== UI Components for YubiKey iOS Apps
+Download the https://github.com/Yubico/yubikit-ios[SDK.zip] file for a list of free-to-use example assets for use in your YubiKey-enabled iOS application. Under the extracted folder: 
```

Comment:
Should this link to https://github.com/Yubico/yubikit-ios/releases which hold the zip files?

Reply:
No, per Lan's email the link is correct.